icelake/src/types/in_memory.rs

```diff
@@ -218,7 +218,9 @@ impl Serialize for PrimitiveValue {
             PrimitiveValue::Long(value) => serializer.serialize_i64(*value),
             PrimitiveValue::Float(value) => serializer.serialize_f32(value.0),
             PrimitiveValue::Double(value) => serializer.serialize_f64(value.0),
-            PrimitiveValue::Decimal(value) => serializer.serialize_str(&value.to_string()),
+            PrimitiveValue::Decimal(value) => {
+                serializer.serialize_bytes(&value.mantissa().to_be_bytes())
```

Comment:
This will lose scale?

Reply:
Scale only affect position of float point and it should be in the type info. But this indeed may cause inconsistence. Maybe we should use i128 in PrimitiveValue::Decimal directly?

Reply:
Using i128 may make things difficult when you want to process it. For example, when you want to print it, you need to carry the data type. I think it's ok for now.

Reply:
How about
```
struct Decimal {
    value: i128,
    type: Arc<DecimalType>
}
```
?

Reply:
This may increase memory usage of decimal type. I'm not sure if it's really worth it. Maybe i128 itself is good enough.